backend/src/main/java/com/example/cms/file/FileResourceController.java

```diff
@@ -25,11 +29,18 @@ public class FileResourceController {
 
     @GetMapping("/all/page/{pageId}")
     public ResponseEntity<List<FileDtoSimple>> getAll(@PathVariable Long pageId, Pageable pageable, @RequestParam Map<String, String> vars) {
+
+        Page<FileResource> responsePage = fileService.getAll(
+                pageable,
+                pageId,
+                FilterPathVariableValidator.validate(vars, FileResource.class));
+
+        HttpHeaders httpHeaders = new HttpHeaders();
+        httpHeaders.set("Whole-Content-Length", String.valueOf(responsePage.getTotalElements()));
```

Comment:
Nie ma czasem jakiegoś wymogu, że custom http header powinien się zaczynać od `X-`?

Reply:
Wymogu nie ma, ale jest to stosowana [konwencja](http://www.httpwatch.com/HTTPgallery/headers/#:~:text=Some%20web%20applications,it%20is%20running.), więc możesz przerobić

Reply:
Oki